i3pystatus/openvpn.py

```diff
@@ -46,6 +47,7 @@ class OpenVPN(IntervalModule):
         ("status_down", "Symbol to display when down"),
         ("status_up", "Symbol to display when up"),
         ("vpn_name", "Name of VPN"),
+        ("openvpn_lastest", "Use lastest openvpn service names"),
```

Comment:
Fair enough, not the best name then! Maybe `openvpn_latest`? Lastest isn't commonly used to describe the most recent.